tests/bgp/test_bgp_slb.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from tests.common import reboot
+from tests.common.helpers.bgp import BGPNeighbor
+from tests.common.dualtor.mux_simulator_control import mux_server_url                                   # lgtm[py/unused-import]
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor  # lgtm[py/unused-import]
+from tests.common.utilities import wait_until
+
+
+pytestmark = [
+    pytest.mark.topology("t0")
+]
```

Comment:
If it is supported on VS setup, suggest to add below mark:

```
pytest.mark.device_type('vs')
```

Reply:
updated, thanks~